content/tag/map.md

```diff
@@ -50,9 +50,13 @@ TagContext tagCtx = TAGGER.currentBuilder().
 
 {{<highlight cpp>}}
 #include "opencensus/tags/tag_key.h"
+#include "opencensus/tags/tag_map.h"
 
-static const opencensus::tags::TagKey key_method =
-                    opencensus::tags::TagKey::Register("method");
+void Get() {
+  static const auto method = opencensus::tags::TagKey::Register("method");
+  opencensus::tags::TagMap tm({{method, "memcache.Client.Get"}});
```

Comment:
as the reading flow from key -> value -> map, it makes sense for user to use the same way to declare value for TagMap.

saying
```
ABSL_CONST_INIT const absl::string_view kGetMethod =
    "memcache.Client.Get";

void Get() {
  static const auto method = opencensus::tags::TagKey::Register("method");
  opencensus::tags::TagMap tm({{method, kGetMethod}});
}
```

and it still makes sense to provide example to use `TagMap::TagMap(std::vector<std::pair<TagKey, std::string>> tags)`, so you can also provide it as 2nd example.

Reply:
Done and done. I also tried compiling it. :)